src/main/java/org/snomed/aag/rest/ServiceIntegrationController.java

```diff
@@ -63,12 +64,30 @@ public ServiceIntegrationController(CommitInformationValidator commitInformation
 
 			return ResponseEntity.status(HttpStatus.OK).build();
 		} else {
-			boolean pacComplete = projectAcceptanceCriteriaService.incrementIfComplete(commitInformation);
+			String sourceBranchPath = commitInformation.getSourceBranchPath();
+			ProjectAcceptanceCriteria projectAcceptanceCriteria = projectAcceptanceCriteriaService.findEffectiveCriteriaWithMandatoryItems(sourceBranchPath);
+			if (projectAcceptanceCriteria == null) {
+				String message = String.format("No Project Acceptance Criteria found for branch %s. Returning %s.", sourceBranchPath, HttpStatus.NO_CONTENT);
+				logger.info(message);
+				return ResponseEntity
+						.status(HttpStatus.NO_CONTENT)
+						.body(message);
+			}
+
+			if (!projectAcceptanceCriteria.hasCriteria()) {
+				String message = String.format("Project Acceptance Criteria on branch %s has no Criteria Items. Returning %s.", sourceBranchPath, HttpStatus.NO_CONTENT);
+				logger.info(message);
+				return ResponseEntity
+						.status(HttpStatus.NO_CONTENT)
+						.body(message);
+			}
+
+			boolean pacComplete = projectAcceptanceCriteriaService.incrementIfComplete(projectAcceptanceCriteria, sourceBranchPath);
```

Comment:
Bulk of the change is here: 

If no PAC configured for a branch or if the PAC has no associated criteria, return 204.